src/containers/booksform.js

```diff
@@ -4,26 +4,31 @@ import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
 import { addBook } from '../actions/index';
 import { FILTERS } from '../constants/constants';
-import Options from './options';
+import Options from '../components/options';
 
 const BookForm = props => {
   const [state, setState] = useState({
     title: '',
     category: FILTERS.action,
   });
   const { getBook } = props;
-  const setTitle = e => setState({ ...state, title: e.target.value });
-  const setCategory = e => setState({ ...state, category: e.target.value });
+  const handleChange = (e, title = false) => {
+    if (title) {
+      setState({ ...state, title: e.target.value });
+    } else {
+      setState({ ...state, category: e.target.value });
+    }
+  };
```

Comment:
[REQUIRED] Even though it is working, we can simplify it and remove the conditional. You can add name attribute to the **input** and **select** fields. Then in the function we can use e.target.name parameter to update the state. For example,
handleChange = (event) => {
const {name, value} = event.target
setState({...state, name: value})
}